### PR TITLE
W 11121840 flex troubleshoot gr

### PR DESCRIPTION
**Pull request by glenn-rodgers-sf**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

**Review by seblaz — COMMENTED** [review]
Good work!

**Review by seblaz — APPROVED** [review]
LGTM!

File: gateway/1.4/modules/ROOT/pages/flex-troubleshoot-requests.adoc [hunk 1]
```diff
@@ -6,12 +6,16 @@ endif::[]
 
 When configuring and running Flex Gateway, you might encounter connection issues. 
 
-You can troubleshoot the request connection between the API upstream and Flex Gateway, as well as the connection between Flex Gateway and the client, by isolating the connections. Isolating Flex Gateway connections allows you to localize the issue.
+You can troubleshoot the outbound connection between the upstream service and Flex Gateway, as well as the inbound connection between Flex Gateway and the client, by making direct requests. Making direct requests isolates Flex Gateway connections allowing you to localize the issue.
```
Comment (hannanelson) [inline gateway/1.4/modules/ROOT/pages/flex-troubleshoot-requests.adoc]:
```suggestion
By making direct requests, you can troubleshoot the outbound connection between the upstream service and Flex Gateway, as well as the inbound connection between Flex Gateway and the client. Making direct requests isolates Flex Gateway connections, enabling you to localize the issue. 
```

File: gateway/1.4/modules/ROOT/pages/flex-troubleshoot-requests.adoc [hunk 2]
```diff
@@ -6,12 +6,16 @@ endif::[]
...
 
-You can troubleshoot the request connection between the API upstream and Flex Gateway, as well as the connection between Flex Gateway and the client, by isolating the connections. Isolating Flex Gateway connections allows you to localize the issue.
+You can troubleshoot the outbound connection between the upstream service and Flex Gateway, as well as the inbound connection between Flex Gateway and the client, by making direct requests. Making direct requests isolates Flex Gateway connections allowing you to localize the issue. 
 
-== Isolate Connection Between the API Upstream and Flex Gateway
-Isolating the connection between the API upstream and Flex Gateway allows you to test for connection issues between the two.
+The following diagram demonstrates the difference between inbound and outbound requests. 
 
-To isolate the connection between the API upstream and Flex Gateway:
+image:flex-inbound-outbound-requests.png[This image describes how inbound connections are between the client and Flex Gateway and outbound connections are between Flex Gateway and the upstream service, align=center]
```
Comment (hannanelson) [inline gateway/1.4/modules/ROOT/pages/flex-troubleshoot-requests.adoc]: No need to mention "image" in the alt text.

Inbound connections between the client and Flex Gateway and outbound connections between Flex Gateway and the upstream service.

File: gateway/1.4/modules/ROOT/pages/flex-troubleshoot-requests.adoc [hunk 3]
```diff
@@ -6,12 +6,16 @@ endif::[]
...
+
+== Outbound Requests Between Flex Gateway and the Upstream Service
+Isolating the connection between the upstream service and Flex Gateway allows you to test for connection between the two.
```
Comment (hannanelson) [inline gateway/1.4/modules/ROOT/pages/flex-troubleshoot-requests.adoc]: allows -> enables?

allow implies permission

File: gateway/1.4/modules/ROOT/pages/flex-troubleshoot-requests.adoc [hunk 4]
```diff
@@ -45,11 +49,11 @@ kubectl -n <namespace> exec -it <pod-name> -- bash
 curl -v <http://upstream.domain>
 ----
 
-== Isolate Connection Between Flex Gateway and the API Upstream
+== Inbound Requests Between the Client and Flex Gateway
 
-Isolating the connection between the Flex Gateway and the client allows you to test if the connection issue is due to a load balancer or another system added between the gateway and the client.
+Isolating the connection between the Flex Gateway and the client allows you to test if the connection issue is due to a load balancer or another system added between Flex Gateway and the client.
```
Comment (hannanelson) [inline gateway/1.4/modules/ROOT/pages/flex-troubleshoot-requests.adoc]: no article with product names.

Comment (hannanelson) [inline gateway/1.4/modules/ROOT/pages/flex-troubleshoot-requests.adoc]:
```suggestion
Isolating the connection between Flex Gateway and the client enables you to test if the connection issue is due to a load balancer or another system added between Flex Gateway and the client.
```

**Review by hannanelson — APPROVED** [review]

**Review by hannanelson — APPROVED** [review]

**Pull request merged**